clerk.gemspec

```diff
@@ -22,10 +22,9 @@ Gem::Specification.new do |gem|
   gem.version       = Clerk::VERSION
   gem.licenses      = ['MIT']
 
-  # gem.add_dependency 'rails', '~> 3'
   gem.add_dependency 'activerecord', '> 3'
```

Comment:
I wonder if we should bump this to `>= 5` due to `, optional: true` on `belongs_to` ?